charts/velero/README.md

```diff
@@ -155,3 +180,7 @@ helm delete <RELEASE NAME> --purge
 ```bash
 helm delete <RELEASE NAME> -n <YOUR NAMESPACE>
 ```
+
+## Chart configuration examples
+
+In [examples folder](./examples/) located some basic scenario for different plugins. It does not include settings that you need make in your cloud provider, see plugin README for details.
```

Comment:
I like this, thanks for adding this! :)

I would add a link here to go straight to the AWS example and to the vsphere as well

Reply:
Done